lock-api/src/main/java/com/palantir/lock/client/TimeLockClient.java

```diff
@@ -133,6 +136,15 @@ public LockResponse lock(LockRequest request) {
         return response;
     }
 
+    @Override
+    public LockResponse lock(LockRequest lockRequest, ClientLockingOptions options) {
```

Comment:
nit: here and elsewhere where that is the correct behaviour feels like `lock(request)` should delegate to `lock(request, ClientLockingOptions.default())` or some such

Reply:
Makes sense here, yes. For the delegating ones I'm a bit less certain that the default logic should live there as well